blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/UpdateStamp.java

```diff
@@ -35,21 +35,32 @@
      * database.
      */
     private boolean dirty;
-    
+
+    private long lastRollupTimestamp;
+
     public UpdateStamp(long timestamp, State state, boolean dirty) {
         setTimestamp(timestamp);
         setState(state);
         setDirty(dirty);
     }
```

Comment:
Would it be feasible to make one constructor call the other, so we don't duplicate anything?